browser/layout/vanilla.html

```diff
@@ -6,6 +6,9 @@
 		<title>{{#title}}{{{this}}}{{/title}}</title>
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<meta name="theme-color" content="#fff1e0">
+		{{#unless @root.flags.speedcurveLux}}
```

Comment:
Shouldn't this be `if`? I know @leggsimon has strong feelings about using a flag in this way.

Reply:
Correct! The snippet below is to unregister the service worker if the flag is off, so `unless` makes sense there.

Reply:
I'm glad you're thinking of me @remybach 🙃

Reply:
@remybach @leggsimon do you know how to negate in Handlebars? If we can, then we don't need `unless` (unless we want to use the empty `if` with it all in the `else` block)

Reply:
IIC you can't negate in normal handlebars, `n-handlebars` adds the unless helper method

Reply:
:boom: what were I thinking... 0 out of 10 for coding effort, Ben! Thanks Remy & Simon ❤️